scio-parquet/src/main/scala/com/spotify/scio/parquet/avro/package.scala

```diff
@@ -122,8 +145,7 @@ package object avro {
    * Enhanced version of [[com.spotify.scio.values.SCollection SCollection]] with Parquet Avro
    * methods.
    */
-  implicit class ParquetAvroSCollection[T : ClassTag]
-  (val self: SCollection[T]) {
+  implicit class ParquetAvroSCollection[T : ClassTag](val self: SCollection[T]) {
```

Comment:
+ `extends AnyVal` and move the context bound to the `saveAsParquetAvroFile`?

Reply:
Now I think about this, it might be better to expose `PCollectionWrapper#ct` so that we can remove the `ClassTag` context bound everywhere. I'll do it in a separate PR.

Reply:
Are you gonna submit that patch before this PR or after?

Reply:
After, I remember why `ct` was protected now, changing it to public would require changing all subclasses. Better do it in a separate one.